src/transaction/components/TransactionProvider.tsx

```diff
@@ -52,17 +53,22 @@ export function useTransactionContext() {
 
 export function TransactionProvider({
   calls,
-  capabilities,
+  capabilities: transactionCapabilities,
   chainId,
   children,
   contracts,
+  isSponsored,
   onError,
   onStatus,
   onSuccess,
 }: TransactionProviderReact) {
   // Core Hooks
   const account = useAccount();
   const config = useConfig();
+  const {
+    config: { paymaster } = { paymaster: undefined },
+  } = useOnchainKit();
+
```

Comment:
with this fix, I should remove https://github.com/coinbase/onchainkit/blob/main/src/nft/components/mint/NFTMintButton.tsx#L37-L39 right?  @0xAlec

Reply:
you'll need to add an `isSponsored` prop to the component and pass it to `Transaction`